src/store/index.js

```diff
@@ -46,4 +47,5 @@ export default class MainStore {
     drawingCursor = new DrawingCursorStore(this);
     chartTable = new ChartTableStore(this);
     lastDigitStats= new LastDigitStatsStore(this);
+    bottomWidgetsContainerStore = new BottomWidgetsContainerStore(this);
```

Comment:
@morteza-binary you can remove the store from end of the object name here. pls just use `bottomWidgetsContainer`